### ISSUE TITLE
[Feature Request] Open links from the repository in a webview inside the application

### ISSUE THREAD

**Issue by douglasaugusto** [mmazzarolo/top-github#6]
Opening the repository links in the mobile device browser and then getting back to the application can be annoying, so I suggest opening the links in a webview of the application itself.

An example of the functionality, is the Facebook application itself.

I hope it is well explained, and in case you need help, I can build on the development of it.

**Comment by mmazzarolo** [comment]
Hello @douglasaugusto, a PR would of course be appreciated 👍
The important bit here is that imho it should be an option in the setting, let me know what do you think about it.  

Sorry for answering a bit late, I was on vacation.

**Comment by douglasaugusto** [comment]
Hi @mmazzarolo 

I agree that this may be an option to be selected, since some users may have Git Clients on their mobile phone and want to open it, for example, not inside the app itself.

Contact that the application still does not have a Settings tab, besides this option to open in or out of the application, do you think valid already leave open a color customization for the application?

If so, I can add this together in PR, and the settings tab gets more than one customization.

**Comment by mmazzarolo** [comment]
Of course, it would be nice touch, but also keep in mind that many smaller PRs are easier to review than a massive one.  
Do you already have any plan on how to implement the "Settings page" in the UI?

**Comment by douglasaugusto** [comment]
I can try to separate the PRs in:
- Creating the configuration
- Creation of the color customization option within the settings
- Creation of the mode of visualization of the links inside the settings

Regarding the location of the settings in the UI:
I think it's cool for Navigation Drawer to remain alone with the selection of languages, so I thought the best place would be to place the icon [more_vert](https://material.io/icons/#ic_more_vert) on the Right side of the current period/calendar selection icon.

What do you think of this approach?

**Comment by mmazzarolo** [comment]
I was thinking the same, 100% on point 👌  

P.S.: I'll obviously mention you as a contributor/author both on the repo and on the Google Play Store as soon as the PRs land.
